README.md

```diff
@@ -39,11 +44,50 @@ discovery in human whole-genome sequencing (WGS) and exome sequencing data.
 - Summary Metrics
 
 ### Software version requirements :
-- GATK 4.beta.5 or later for PairedSingleSampleWF. GATK4.0.1.0 or later for JointGenotypingWf.
-- Picard 2.x
-- Samtools (see gotc docker)
+- GATK 4.beta.5 for PairedSingleSampleWF. GATK4.0.11.0 for JointGenotypingWf.
```

Comment:
Why take out the "or later"?

Reply:
The WDL uses this [gotc docker](https://github.com/broadinstitute/dsde-pipelines/blob/44cf77a4aded80fb62821f5e72cb7176cb30fa69/genomes_in_the_cloud/single_sample/PairedSingleSampleWf.options.json#L6) which only uses gatk4.beta.5. This beta version still uses camelcase for its tool parameter while the released version uses dashes. This wdl uses camelcase when executing gatk commands so using gatk4 (not beta) would cause an error.